cloudpickle/cloudpickle.py

```diff
@@ -238,7 +238,11 @@ def dump(self, obj):
 
     def save_memoryview(self, obj):
         """Fallback to save_string"""
-        Pickler.save_string(self, str(obj))
+        try:
+            arg = obj.obj
+        except ValueError:
+            arg = _released_memory_view_sentinel
+        self.save_reduce(_fill_memoryview, (arg,), obj=obj)
```

Comment:
This ignores other properties of the memoryview. For example it incorrectly works with `memoryview(b'abcd')[2:]` or `memoryview(b'abcd')[::2]`.